### PR TITLE
V3.62 design consistency on gutenberg blocks

### PR DESCRIPTION
**Pull request by YauheniKapliarchuk**
@designerbrent 

Please review this PR.

Thanks

### CODE REVIEW

File: src/blocks/media-block/editor.scss [hunk 1]
```diff
@@ -68,37 +67,30 @@
...
-        line-height: 1.6;
-        color: #717171;
       }
     }
 
     &-meta__buttom {
-      display: flex;
       justify-content: flex-start;
       flex-wrap: wrap;
       text-transform: uppercase;
```
Comment (designerbrent) [inline src/blocks/media-block/editor.scss:80]: Remove this line.

File: src/blocks/media-block/editor.scss [hunk 2]
```diff
@@ -68,37 +67,30 @@
...
 
       &-description {
-        margin-top: 0.3rem;
-        font-size: 1rem;
-        line-height: 1.6;
-        color: #717171;
       }
     }
 
     &-meta__buttom {
```
Comment (designerbrent) [inline src/blocks/media-block/editor.scss]: This is misspelled. It should be spelled `__bottom` and needs to be corrected in all locations.

File: src/blocks/media-block/components/MediaBlockEditComponent.js [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+                                            <RichText
+                                                className={" contentCard__input"}
+                                                placeholder={ __("Enter your Description here...") }
+                                                value={ attributes.description }
+                                                onChange={ this.onChangeDescription }
+                                            />
+                                        </fieldset>
+                                    </div>
+                                </div>
+                                <div className={"alps__media-block__text-section-meta__buttom"}>
```
Comment (designerbrent) [inline src/blocks/media-block/components/MediaBlockEditComponent.js]: Correct classname to `alps__media-block__text-section-meta__bottom`.

File: src/blocks/media-block/components/MediaBlockEditComponent.js [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+                                    <div className={"alps__media-block__text-section-meta__buttom__category"}>
```
Comment (designerbrent) [inline src/blocks/media-block/components/MediaBlockEditComponent.js]: Correct classname to `alps__media-block__text-section-meta__bottom__category`

File: src/common.scss [hunk 5]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+
+.descCard {
+  height: 70px;
+  background: #F3F4F6;
+  border-radius: 4px 4px 0px 0px;
+
+  display: flex;
+  //padding: 23px 0 23px 26px;
+
+  font-family: Avenir;
```
Comment (designerbrent) [inline src/common.scss]: Remove

File: src/common.scss [hunk 6]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+
+  &__info-icon-box {
+    padding-top: 23px;
+    padding-bottom: 24px;
+  }
+
+  &__info-icon {
+    padding-right: 8px;
+    display: inline;
+  }
```
Comment (designerbrent) [inline src/common.scss]: Add style: 
`opacity: 0.5;`

File: src/common.scss [hunk 7]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+
+  fieldset {
+    display: flex;
+    border-top: 1px solid #D2D9E4;
+    margin-bottom: 10px;
+    padding-left: 0;
+  }
+
+  legend {
+    font-family: Avenir, normal;
```
Comment (designerbrent) [inline src/common.scss]: Remove all `font-family` declarations.

File: src/common.scss [hunk 8]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+    font-weight: 800;
+    line-height: 135.7%;
+    font-size: 18px;
+    color: rgba(25, 37, 62, 0.7);;
+    margin-left: 20px;
```
Comment (designerbrent) [inline src/common.scss]: Switch to `0`.

File: src/common.scss [hunk 9]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+    padding: 0 15px 0 15px;
```
Comment (designerbrent) [inline src/common.scss]: Switch to `0 15px 0 0`

File: src/common.scss [hunk 10]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+  &__legend-cta {
+    font-size: 14px;
+  }
+
+  .components-text-control__input {
+    border: none;
+    border-bottom: 1px solid #CCD5DE;
+    padding-left: 0;
+
+    font-family: Avenir;
```
Comment (designerbrent) [inline src/common.scss]: Remove all `font-family` declarations.

File: src/common.scss [hunk 11]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+
+  .components-text-control__input::placeholder {
+    color: rgba(25, 37, 62, 0.7);
+  }
+
+  //Override
+  .block-editor-rich-text__editable {
+    padding-left: 0;
+
+    font-family: Avenir;
```
Comment (designerbrent) [inline src/common.scss]: Remove all `font-family` declarations.

File: src/common.scss [hunk 12]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+  }
+
+  //Override
+  .button {
+    border: 1.5px solid #6C96E7;
+    box-sizing: border-box;
+    border-radius: 8px;
+    background-color: white;
+
+    font-family: Avenir;
```
Comment (designerbrent) [inline src/common.scss]: Remove all `font-family` declarations.

File: src/common.scss [hunk 13]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+    box-sizing: border-box;
+    border-radius: 2px;
+  }
+  .components-checkbox-control__input:checked {
+    background: #3E4E6C;
+    border-radius: 2px;
+  }
+
+  .components-checkbox-control__label {
+    font-family: Avenir;
```
Comment (designerbrent) [inline src/common.scss]: Remove all `font-family` declarations.

File: src/common.scss [hunk 14]
```diff
@@ -80,10 +80,200 @@ figure.is-selected .components-button.blocks-gallery-item__remove,
...
+  fieldset {
+    display: flex;
+    border-top: 1px solid #D2D9E4;
+    margin-bottom: 10px;
+    padding-left: 0;
+  }
+
+  legend {
+    font-family: Avenir, normal;
+    font-weight: 800;
```
Comment (designerbrent) [inline src/common.scss]: Switch to `600`.

File: src/blocks/image-breakout/style.scss [hunk 15]
```diff
@@ -4,11 +4,12 @@
...
-  @media (min-width: 700px) {
-    margin-left: calc((100vw / 7) * -1);
-  }
-}
+//
+//.wp-block-alps-gutenberg-blocks-image-breakout {
+//  @media (min-width: 700px) {
+//    margin-left: calc((100vw / 7) * -1);
+//  }
+//}
```
Comment (designerbrent) [inline src/blocks/image-breakout/style.scss]: Why is this getting commented out? This needs to be here to prevent gaps like this: 

2021-05-03 at 11 59 AM

**Review by designerbrent — CHANGES_REQUESTED** [review]
## Accordion
### Placeholder Text
Please change the placeholder text to the following:
- Title: `Enter your Title...`
- Body: `Enter your Body Text...`



## Blockquote
### Control block: 
Please make sure the margin on the Extend Quote and Strong Quote controls have a left margin like the Paragraph block does.
2021-05-03 at 10 33 AM

## Content Block
Why does the `Link URL` have a block around it? the other input are all with an underline but the link has a box around it.
2021-05-03 at 10 39 AM

### Placeholder Text
Please change the placeholder text to the following:
- Title: `Enter your Title...`
- Read More Button: `Default: Read More`
- Link URL: `https://...`


## Content Expand
Nothing has been done yet to this block.

## Content Show More
### Placeholder Text
Please change the placeholder text to the following:
- Title: `Enter your Title...`
- Description: `Enter your Description...`
- Body: `Enter your Body Text... (Displays on click for Show More button)
- 

## CTA 
### Card Title
Please change card title to `Call to Action (CTA)`. But keep the other references to CTA.

### Control block: 
Please make sure the margin on the Dark Background and Background Image controls have a left margin like the Paragraph block does.
2021-05-03 at 11 01 AM

### Buttons: 
Reverse the order of the button fields, so it should start with the Text input on top followed by the URL input.

### Placeholder Text
Please change the placeholder text to the following:
- Title: `Enter your Title...`
- Description: `Enter your Description...`
- Body: `Enter your Body Text... (Displays on click for Show More button)
- Button 1 Text: `Button Label`
- Button 1 URL: `https://...`
- Button 2 Text: `Button Label`
- Button 2 URL: `https://...`
- Both open checkboxes: `Open link in a new window`

## Gallery
### Default View
Do we need to have the second "Gallery" title? 
2021-05-03 at 11 06 AM

### Placeholder Text
Please change the placeholder text to the following:
- Gallery Title: `Enter your Gallery Title...`

## Highlighted Paragraph
### Placeholder Text
Please change the placeholder text to the following:
- Content: `Enter your Content...`

## Image (2UP)
### Default View
Do we need to have the second "Image (2UP)" title? 
2021-05-03 at 11 08 AM

### Images Selected
Only 1 image is showing on the admin side.
1. Selected 2 images
2021-05-03 at 11 10 AM
2. Only 1 shows.
2021-05-03 at 11 11 AM

The description is just showing the HTML, not rendering it.

## Image Breakout
### Placeholder Text
Please change the placeholder text to the following:
- Caption: `Enter your Caption...`

## Inline Sidebar
### Placeholder Text
Please change the placeholder text to the following:
- Preface: `Enter your title Preface...`
- Title: `Enter your Title...`

## Latest Posts

### Posts with no images
If there are no images on a post, the width of the text should be 3 columns, not 2. The functionality of the block in the current release is working correct.
2021-05-03 at 12 03 PM


### Placeholder Text
Please change the placeholder text to the following:
- Title: `Enter your Title...`
- Link Label: Please change this title to `See All Link Label`
- Link Label: `Enter your link label...`
- See All URL: `https://...`

## Media Block
### Placeholder Text
Please change the placeholder text to the following:
- Title: `Enter your Title...`
- Description: `Enter your Description...`
- Category: `Enter your Categories...`
- Date: `Enter your Date...`

## Media Testimonies
### Placeholder Text
Please change the placeholder text to the following:
- Title: `Enter your Title...`
- Quote: `Enter your Quote...`
- Video: `Enter your Video title...`

**Comment by designerbrent** [comment]
A couple other items that came up on our call today that need to be addressed: 

- [x] **Repeated Styles**: In the final rendered `.css`, all the styles are repeated several times. This needs to be fixed.
- [x] **Rich text inputs**: If we can't control which fields have rich text controls, at least make the output work when they do. The button labels currently are outputting plain text, and showing the HTML, not rendering it.
- [x] **Upload Button** Please change the color of the upload button to match the dark blue. You can use the lightblue on hover, if you want, but I want to see the button be the same as the other text labels.

File: src/blocks/media-block/editor.scss (hunk 1 above)
Comment (YauheniKapliarchuk) [reply src/blocks/media-block/editor.scss:80]: It was removed

File: src/blocks/media-block/editor.scss (hunk 2 above)
Comment (YauheniKapliarchuk) [reply src/blocks/media-block/editor.scss]: I was fixed. Thanks

File: src/blocks/media-block/components/MediaBlockEditComponent.js (hunk 3 above)
Comment (YauheniKapliarchuk) [reply src/blocks/media-block/components/MediaBlockEditComponent.js]: It was fixed. Thanks

File: src/blocks/media-block/components/MediaBlockEditComponent.js (hunk 4 above)
Comment (YauheniKapliarchuk) [reply src/blocks/media-block/components/MediaBlockEditComponent.js]: It was fixed. Thanks

File: src/common.scss (hunk 5 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 6 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 9 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 14 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 7 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 10 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 11 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 12 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 13 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/common.scss (hunk 8 above)
Comment (YauheniKapliarchuk) [reply src/common.scss]: It was fixed. Thanks

File: src/blocks/image-breakout/style.scss (hunk 15 above)
Comment (YauheniKapliarchuk) [reply src/blocks/image-breakout/style.scss]: It was fixed. Thanks

**Review by designerbrent — CHANGES_REQUESTED** [review]
## Items to Address


Here are my comments regarding the pull request. While a lot of the placeholder text was done, many of the other items were not addressed. Please addressed the individual check boxes for each block.

## Global Items: 

- [x] **Upload Image** button: Edit the colors to match the blue of the headlines.  2021-05-10 at 7 18 PM


## Blockquote

- [x] Please make sure the margin on the Extend Quote and Strong Quote controls have a left margin like the Paragraph block does. 2021-05-03 at 10 33 AM

## Content Block
- [x] Can you disable the editor blocks for the button fields? 2021-05-10 at 7 15 PM

## Content Expand

- [x] Please finish upgrading this block. This one has been around longer and needs to be upgraded.

## Content Show More
- [x] The `x` button is missing from the image to remove it. 2021-05-10 at 7 21 PM


## Call to Action (CTA)
- [x] **Title:** Please change card title to `Call to Action (CTA).` But keep the other references to `CTA`. (I want to make sure the existing blocks still work.)
- [x] **Control Block:** Please make sure the margin on the Dark Background and Background Image controls have a left margin like the Paragraph block does. 2021-05-03 at 11 01 AM

## Image (2UP)
- [x] Negative margin should **ONLY** apply on the front side, not the edit side. 2021-05-10 at 7 25 PM
- [ ] Negative Margin on the front shouldn't start at `701px`, not `1301px`

## Image: Breakout
- [x] Negative margin is not applying from `701px` and wider. 2021-05-10 at 7 37 PM

## Inline Block
- [x] Since this block is a duplicate of the `Content Expand` block, please remove this from the code.


<hr /> 

# Completion List

(@YauheniKapliarchuk please don't check off the items. This is for Brent to check off as he's done reviewing each item.)

- [x] Accordion
- [x] Blockquote
- [ ] Content Block
- [x] Content Expand
- [x] Content Show More
- [x] CTA 
- [x] Gallery
- [x] Highlighted Paragraph
- [x] Image (2UP)
- [x] Image Breakout
- [x] Inline Sidebar
- [x] Latest Posts
- [x] Media Block
- [x] Media Testimonies

**Review by designerbrent — CHANGES_REQUESTED** [review]
Just a few more changes that go missed: 

## Global Items: 

- [x] Upload Image button: Edit the colors to match the blue of the headlines. 2021-05-11 at 4 26 PM

## Content Block

- [x] When HTML is used in the **Read More** button, the HTML and content is stripped. Please remove the HTML only, not that content inside the markup.

## Content Expand
- [x] The backend works fine, but on the frontend, the Title doesn't show and the Body is pulling content from somewhere else.

**Comment by designerbrent** [comment]
## Inline Sidebar:

- [x] Please remove the reference to the Inline Sidebar so the block will compile. https://github.com/adventistchurch/alps-gutenberg-blocks/blob/ce127982d7ae83f1e95fed69030c9b178a6cffd3/src/blocks.js#L22

**Review by designerbrent — APPROVED** [review]
I see that you found a way to disable the bold, italic, etc for the buttons. 👏 

💯

**Pull request merged**